src/main/java/org/breedinginsight/brapi/v2/BrAPIStudiesController.java

```diff
@@ -74,7 +74,7 @@ public BrAPIStudiesController(BrAPIStudyService studyService, StudyQueryMapper s
 
     @Get("/studies{?queryParams*}")
     @Produces(MediaType.APPLICATION_JSON)
-    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.ALL})
+    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.PROGRAM_SCOPED_ROLES})
     public HttpResponse<Response<DataResponse<List<BrAPIStudy>>>> getStudies(
```

Comment:
Ultimately this method needs to allow Experimental Collaborator role access. You can either do that in this PR or I will make the change in BI-2258 (where I have also implemented filtering in this method).

Reply:
The `roles` now includes `ProgramSecuredRole.EXPERIMENTAL_COLLABORATOR`